usaspending_api/broker/helpers/get_business_categories.py

```diff
@@ -416,3 +416,429 @@ def get_business_categories(row, data_type):
             "Invalid object type provided to update_business_categories. "
             "Must be one of the following types: TransactionFPDS, TransactionFABS"
         )
+
+
+def get_business_categories_fabs(business_types):
```

Comment:
Wow. 
How did this used to be done? Postgres function? Or done just in Python because it was RBAR'ing 1 row at a time (possibly from a chunked row-set)?